hack/generate-parallel.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CURRENT_DIR="$(dirname $0)"
+PROJECT_ROOT="${CURRENT_DIR}"/..
+if [ "${PROJECT_ROOT#/}" == "${PROJECT_ROOT}" ]; then
+  PROJECT_ROOT="./$PROJECT_ROOT"
+fi
+
+pushd "$PROJECT_ROOT" > /dev/null
+ROOTS=${ROOTS:-$(git grep --files-with-matches -e '//go:generate' cmd extensions pkg plugin test | \
+	xargs -n 1 dirname | \
+	sed 's,^,github.com/gardener/gardener/,;' | \
+	sort | uniq
+)}
+popd > /dev/null
+
+read -ra PACKAGES <<< $(echo ${ROOTS})
+
+GO111MODULE=on parallel go generate -mod=vendor {} ::: ${PACKAGES[@]}
```

Comment:
Add some info how to install this tool in https://github.com/gardener/gardener/blob/master/docs/development/local_setup.md. You can also add it to https://github.com/gardener/gardener/blob/master/hack/install-requirements.sh.

If you enable the CI/CD to benefit from the parallel execution, you also have to double check if it is required here https://github.com/gardener/gardener/blob/master/.ci/verify#L13

Reply:
I will do this, I intentionally didn't bother at this stage since I wanted to collect feedback first.